api/src/controllers/profile.ts

```diff
@@ -105,16 +107,22 @@ export const updateProfileContacts = async (
 
     const provisionerRelatedChanges = editCompares.some(editCompare => editCompare);
     if (provisionerRelatedChanges) {
-      await requestProfileContactsEdit(Number(profileId), contacts);
+      const request = await requestProfileContactsEdit(Number(profileId), contacts, user);
+      await fulfillRequest(request);
       res.status(202).end();
     } else {
+      const request = await requestProfileContactsEdit(Number(profileId), body, user);
```

Comment:
For the `else` above I tend to prefer early returns now days rather than `else` where applicable.
```javascript
return res.status(202).end();
```

Not always great as the TypeScript compiler may complain about inconsistent returns but I think you get the idea.